### DISCUSSION TITLE
InputNumber onValueChange ignores Numpad Enter

### DISCUSSION THREAD

**Discussion by SergiuCimpoi** [primefaces/community#1379] [PrimeReact] [answered]
One of the common use cases when you type numbers is to use the numpad keys, which naturally leads to validating the input by pressing the numpad Enter key, out of obvious reasons. 
The InputNumber component validates the model on Tab and Enter only, NumpadEnter case missing.
An extra _case 'NumpadEnter''_ would do wonders for almost no costs.

**Comment by melloware** [comment, marked as answer]
I created an issue for you: https://github.com/primefaces/primereact/issues/6107
